README.md

```diff
@@ -40,7 +51,7 @@ The named parameters are as follows:
 
 ##### --private-key (for authentication using an Adobe I/O integration)
 
-The local path (relative or absolute) to the RSA private key. Instructions on how to generate this key can be found in the [Getting Started guide](https://developer.adobelaunch.com/guides/extensions/getting-started/) and should have been used when creating your integration through the Adobe I/O console.
+The local path (relative or absolute) to the RSA private key. Instructions on how to generate this key can be found in the [Getting Started guide](https://developer.adobelaunch.com/api/guides/access_tokens/#authentication-for-each-session) and should have been used when creating your integration through the Adobe I/O console.
```

Comment:
@chip902 I think this works for now but ultimately we want to [generate access tokens programmatically](https://developer.adobe.com/developer-console/docs/guides/authentication/ServerToServerAuthentication/implementation/#generating-access-tokens-programmatically) if possible

Reply:
100%.  I think I'll keep this though for now but then add a new section for this after.  I personally think its important to show the end user how the script works stand alone before intoducing anything programatic or automation